src/main/java/core/basesyntax/service/impl/CalculateServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.StorageTransaction;
+import core.basesyntax.service.CalculateService;
+import core.basesyntax.strategy.TransactionActivity;
+import java.util.List;
+
+public class CalculateServiceImpl implements CalculateService {
+    private final TransactionActivity activity = new TransactionActivity();
```

Comment:
It is better to use the constructor
```suggestion
    private final TransactionActivity activity;
```